ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-maistra-2.4.yaml

```diff
@@ -20,6 +20,51 @@ tests:
     make lint
   container:
     from: src
+- as: integ-make-test-2-4
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h0m0s
+    version: "4.10"
+  steps:
+    env:
+      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:2.4
+      MAISTRA_NAMESPACE: maistra-e2e-test
+      MAISTRA_SC_POD: maistra-e2e-test-sc-pod
+    test:
+    - as: copy-src
+      cli: latest
+      commands: |
+        # SRC_PATH does end with /. : the content of the source directory is copied into dest directory
+        oc cp ./. "${MAISTRA_NAMESPACE}"/"${MAISTRA_SC_POD}":/work/
+      env:
+      - name: MAISTRA_NAMESPACE
+      - name: MAISTRA_SC_POD
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 100Mi
+      timeout: 20m0s
+    - as: make-test-2-4
+      cli: latest
+      commands: |
+        oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" \
+          go install github.com/jstemmer/go-junit-report/v2@latest
```

Comment:
The builder image has it.

If that's not enough, tweak the image, or leave this to the test.

Reply:
ok, I think it's a GOPATH issue. I will tweak the image later.